contracts/contracts/mocks/MockRebornMinter.sol

```diff
@@ -97,15 +97,15 @@ contract Reborner {
         log("We are attempting to mint..");
         address asset = sanctum.asset();
         address vault = sanctum.vault();
-        IERC20(asset).approve(vault, 1e18);
-        IVault(vault).mint(asset, 1e18, 0);
+        IERC20(asset).approve(vault, 1e6);
+        IVault(vault).mint(asset, 1e6, 0);
         log("We are now minting..");
     }
 
     function redeem() public {
         log("We are attempting to redeem..");
         address vault = sanctum.vault();
-        IVault(vault).redeem(1e18, 1e18);
+        IVault(vault).redeem(1e18, 0);
```

Comment:
would it make sense to enforce at least some `_minimumUnitAmount` when redeeming?